### PR TITLE
Create 115.com.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

File: src/chrome/content/rules/115.com.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!--
+
+	Mismatch:
+		tj.data
+
+	MCB:
+		help
+
+	Redirect to http:
```
Comment (J0WI) [inline src/chrome/content/rules/115.com.xml]: So why are they included in the list of targets?

File: src/chrome/content/rules/115.com.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	<target host="m.115.com" />
+	<target host="my.115.com" />
+	<target host="msg.115.com" />
+	<target host="note.115.com" />
+	<target host="passport.115.com" />
+	<target host="pay.115.com" />
+	<target host="pc.115.com" />
+	<target host="q.115.com" />
+	<target host="static.115.com" />
+	<target host="thumb.115.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/115.com.xml]: vip.115.com

File: src/chrome/content/rules/115.com.xml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	Mismatch:
+		tj.data
+
+	MCB:
+		help
+
+	Redirect to http:
+		assets.115img.com
+		static.115img.com
+
```
Comment (J0WI) [inline src/chrome/content/rules/115.com.xml]: Got a timeout on tool.115.com

File: src/chrome/content/rules/115.com.xml (hunk 1 above)
Comment (renyouguo) [inline src/chrome/content/rules/115.com.xml]: @J0WI So what should I do? Delete this note?

**Pull request merged**